docs/horizen_eon/05-staking/05-faq.md

```diff
@@ -10,31 +10,188 @@ To stop forging, call the endpoint `/block/stopForging`.
+`
+
+#### Check for peers
+
+`docker exec evmapp curl -sX POST "http://127.0.0.1:9545/node/connectedPeers" -H "accept: application/json"`
+
+#### Check block height
+
+`docker compose exec evmapp gosu user curl -sX POST "http://127.0.0.1:9545/block/best" -H "accept: application/json" | jq '.result.height'`
+
+#### Check for banned peers
+
+`docker exec evmapp gosu user bash -c 'curl -X POST "http://127.0.0.1:${SCNODE_REST_PORT}/node/blacklistedPeers" -H  "accept: application/json"'`
+
+#### Blockchain is stuck syncing
+
+If the blockchain is totally stuck, or corrupted, delete the existing blockchain on the forger and restart the syncing:
+in compose-evm-simplified/deployments/forger/eon directory:
+
+`docker compose -f docker-compose.yml down evmapp`
+
+`docker volume ls`
+
+`docker volume rm evmapp-mainnet-eon_evmapp-data`
+
+`docker volume ls`
+
+`docker compose restart evmapp`
+
+Then, in /compose-evm-simplified directory:
+
+`./scripts/forger/generate_keys.sh`
+
+#### Check your node is available to connect by scanning it from another machine
+
+`nmap 4.16.58.254 -p 9084`
+
+Which should show a result of:
+
+>Starting Nmap 7.94 ( https://nmap.org ) at 2024-03-14 09:36 EDT
+>
+>Nmap scan report for 4.16.58.254
+>
+>Host is up (0.016s latency).
+>
+>PORT     STATE SERVICE
+>
+>9084/tcp open  aurora
+
+
+Scan your forger from an external machine ports 1-1000
+
+>nmap 4.16.58.254 -p 1-10000
+>
+>Starting Nmap 7.60 ( https://nmap.org ) at 2024-03-14 09:39 EDT
+>
+>Nmap scan report for 4.16.58.254
+>
+>Host is up (0.00070s latency).
+>
+>Not shown: 9996 closed ports
+>
+>PORT     STATE SERVICE
+>
+>22/tcp   open  ssh
+>
+>9033/tcp open  unknown
+>
+>9084/tcp open  aurora
+>
+>Nmap done: 1 IP address (1 host up) scanned in 0.31 seconds
+
+
+### Logs
+
+#### Check the evmapp logs with:
+
+`docker logs --tail 60 evmapp`
+
+#### If the forger does not forge a block (most of the time):
+
+>[INFO ] 2024-03-15 09:43:40:450 +0000 [ AbstractForger.scala:176 ] [scala-execution-context-global-72] io.horizen.account.forger.AccountForger - Slot is skipped with reason: No eligible forging stake found.
+
+#### If you forge the block, the log will say this instead:
+
+>[INFO ] [ AbstractForger.scala:170 ] [scala-execution-context-global-51657] io.horizen.account.forger.AccountForger - Got successfully forged block with id BLOCKHASH
+
+
+#### Check that the forger is forging blocks:
+
+`docker logs evmapp | grep forged`
+
+or
+
+`cat logs/evmapp/debug.log | grep -e 'Successfully generated block'`
+
+### Working with Zend
+
+#### Check the zend logs with
+
+`docker logs --tail 60 zend`
+
+#### Get shell on zend container:
+
+`docker exec -it zend /bin/bash`
+
+### Useful tags for the evmapp .env file
+
+This is located in: compose-evm-simplified/deployments/forger/eon
+
+#### Legacy CPU
+
+`ZEND_TAG=v5.0.1-legacy-cpu`
```

Comment:
Remove this, EON doesn't work with legacy CPUs, even though zend would work with old CPUs, it wouldn't help users as they wouldn't be able to run EON.